src/main/java/gregtech/common/metatileentities/multi/steam/MetaTileEntitySteamOven.java

```diff
@@ -58,16 +57,11 @@ protected BlockPattern createStructurePattern() {
                 .aisle("XXX", "CCC", "#C#")
                 .aisle("XXX", "C#C", "#C#")
                 .aisle("XXX", "CSC", "#C#")
-                .setAmountAtLeast('L', 6)
-                .setAmountAtMost('H', 1)
                 .where('S', selfPredicate())
-                .where('L', statePredicate(getCasingState()))
-                .where('H', abilityPartPredicate(MultiblockAbility.STEAM))
-                .where('X', state -> statePredicate(getFireboxState())
-                        .or(abilityPartPredicate(MultiblockAbility.STEAM)).test(state))
-                .where('C', statePredicate(getCasingState()).or(abilityPartPredicate(
-                        MultiblockAbility.STEAM_IMPORT_ITEMS, MultiblockAbility.STEAM_EXPORT_ITEMS)))
-                .where('#', (tile) -> true)
+                .where('X', states(getFireboxState()).setMinGlobalLimited(6).or(autoAbilities(true, false, false, false)))
```

Comment:
In the previous pattern, there was not a limit on the `getFireboxState()` entries, and this is also missing the maximum limit of 1 for the Steam Ability.

Reply:
Steam Ability was limited in the autoAbilities.